src/llmcompressor/transformers/finetune/session_mixin.py

```diff
@@ -39,7 +39,7 @@
 ]
 
 TRAINER_STATE_NAME = "trainer_state.json"
-METADATA_ARGS = [
+METAdataset_args = [
```

Comment:
this should probably not change?
```suggestion
METADATA_ARGS = [
```

Reply:
thanks!